packages/storage-ui/src/Components/Modules/FilesList/FilesList.tsx

```diff
@@ -799,6 +800,11 @@ const FilesList = () => {
                     <Trans>Date uploaded</Trans>
                   </TableHeadCell>
                 }
+                {
+                  desktop && <TableHeadCell>
+                    CID
+                  </TableHeadCell>
+                }
```

Comment:
Same here (thanks lgtm-bot bis)
```suggestion
                <TableHeadCell>
                    CID
                </TableHeadCell>
```